public/app/features/dashboard/components/AnnotationSettings/AnnotationSettingsList.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+import { DeleteButton, Icon, IconButton } from '@grafana/ui';
+import EmptyListCTA from 'app/core/components/EmptyListCTA/EmptyListCTA';
+import { DashboardModel } from '../../state/DashboardModel';
+import { ListNewButton } from '../DashboardSettings/ListNewButton';
+import { arrayUtils } from '@grafana/data';
+
+type Props = {
+  dashboard: DashboardModel;
+  onNew: () => void;
+  onEdit: (idx: number) => void;
+};
+
+export const AnnotationSettingsList: React.FC<Props> = ({ dashboard, onNew, onEdit }) => {
+  const [annotations, updateAnnotations] = useState(dashboard.annotations.list);
+
+  const onMove = (idx: number, direction: number) => {
+    dashboard.annotations.list = arrayUtils.moveItemImmutably(annotations, idx, idx + direction);
+    updateAnnotations(dashboard.annotations.list);
+  };
+
+  const onDelete = (idx: number) => {
+    dashboard.annotations.list = [...annotations.slice(0, idx), ...annotations.slice(idx + 1)];
+    updateAnnotations(dashboard.annotations.list);
+  };
+
+  const isEmptyList = annotations.length === 0 || (annotations.length === 1 && annotations[0].builtIn);
+
+  if (isEmptyList) {
+    return (
+      <EmptyListCTA
+        onClick={onNew}
+        title="There are no custom annotation queries added yet"
+        buttonIcon="comment-alt"
+        buttonTitle="Add annotation query"
+        infoBoxTitle="What are annotation queries?"
+        infoBox={{
+          __html: `<p>Annotations provide a way to integrate event data into your graphs. They are visualized as vertical lines
+          and icons on all graph panels. When you hover over an annotation icon you can get event text &amp; tags for
+          the event. You can add annotation events directly from grafana by holding CTRL or CMD + click on graph (or
+          drag region). These will be stored in Grafana's annotation database.
+        </p>
+        Checkout the
+        <a class='external-link' target='_blank' href='http://docs.grafana.org/reference/annotations/'
+          >Annotations documentation</a
+        >
+        for more information.`,
+        }}
+      />
+    );
+  }
+
+  return (
+    <>
```

Comment:
```suggestion
```

Reply:
these are required, cannot return two elements otherwise

Reply:
True, missed that there is a button component at the bottom.